f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_sensitive_information.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?><database>
     <element label="Sensitive Information" max_length="40" name="x_g_dis_atat_sensitive_information" type="collection">
+        <element label="508AccessibilityRequirements" max_length="1000" name="accessibilityrequirements" type="string_full_utf8"/>
```

Comment:
Is there a reason the field type needs to be `String (Full UTF-8)`? If not, I'd suggest sticking with a regular `String` type like the other fields. 

Did this replace the previous `Section 508 Sufficient` field or should both that field and the `508AccessibilityRequirements` be present? The `Section 508 Sufficient` is being used by a flow downstream when creating one of the documents and this would cause that information to no longer being present on the document.

![FieldType_AT-7377](https://user-images.githubusercontent.com/89867104/166505864-301fcbe3-210e-41d6-9d04-a9bf6825c6c9.png)
![Screen Shot 2022-05-03 at 12 48 36 PM](https://user-images.githubusercontent.com/89867104/166501264-94aa0af8-5078-4ddd-958f-55eb9e074fa6.png)
![FlowInformation_AT-7377](https://user-images.githubusercontent.com/89867104/166504660-1bcd11ba-31f7-4bed-937b-e5af60fd64aa.png)


Was the naming for the *Column Name* done that way intentionally? If not, I would also suggest naming the *Column label* as `Accessibility Requirements 508` (numbers are at the beginning are not captured in the name) so the column name ends up being something like `accessibility_requirements_508` for readability and clarity when trying to access that column in a record. You may have to delete the column and make a new one. 
![NewColumn_AT-7377](https://user-images.githubusercontent.com/89867104/166505330-08b841ea-af2b-4a26-8050-ce95a6db67ad.png)

Reply:
This field does not replace the `Section 508 Sufficient` field. If false is selected, it will be the next input on the following page. i have also deleted that column and created a new one with easier readability.